src/PkgDeps.jl

```diff
@@ -164,28 +164,20 @@ function users(
     return downstream_dependencies
 end
 
+function users(pkg_name::String, pkg_registry_name::String=GENERAL_REGISTRY; kwargs...)
+    uuid = _get_pkg_uuid(pkg_name, pkg_registry_name)
+    return users(uuid; kwargs...)
+end
 
+# Useful for testing using with registries not in `DEPOT_PATH`.
 """
-    users(pkg_name::String; pkg_registry_name::String="$GENERAL_REGISTRY")
-
-Find all packages which use `pkg_name`. Use the `pkg_registry_name` to look up the UUID of `pkg_name`.
+    users(pkg_name::String, pkg_registry::RegistryInstance; kwargs...)
 
-# Arguments
-- `pkg_name::String`: Find users of this package
-
-# Keywords
-- `registry_name::String="$GENERAL_REGISTRY"`: Name of registry where `pkg_name` is active
-
-# Returns
-- `Array{String}`: All packages which are dependent on `pkg_name`
+Find the users of the package named `pkg_name` which is registered in `pkg_registry`.
 """
-function users(
-    pkg_name::String;
-    pkg_registry_name::String=GENERAL_REGISTRY,
-    kwargs...
-)
-    uuid = _get_pkg_uuid(pkg_name, pkg_registry_name; kwargs...)
+function users(pkg_name::String, pkg_registry::RegistryInstance; kwargs...)
+    uuid = _get_pkg_uuid(pkg_name, pkg_registry)
```

Comment:
We would still want to pass in `kwargs...` so the user can pass in `depots` to `_get_pkg_uuid()`?

Reply:
No `_get_pkg_uuid` can now take a RegistryInstance too

Reply:
Ah right, missed that!